### PR TITLE
Add debug option to view table related from all badger tables

### PR DESCRIPTION
**Pull request by sana-jawad**
In Debug -> Query Data Store page, there is an option to search only by particular table. In order to make debugging easier, adding an option to search in all the tables and give a complete picture which can help in debugging different issues. Also in the same page the encoded value used to update the text box so fixed that as well.

### CODE REVIEW

File: pkg/sloop/webserver/debug.go [hunk 1]
```diff
@@ -124,32 +124,44 @@ func listKeysHandler(tables typed.Tables) http.HandlerFunc {
...
-				totalCount++
-				thisKey := string(itr.Item().Key())
-				if keyRegEx.MatchString(thisKey) {
-					keys = append(keys, thisKey)
-					count += 1
-					totalSize += itr.Item().EstimatedSize()
-					if count >= maxRows {
-						glog.Infof("Number of rows : %v has reached max rows: %v", count, maxRows)
-						break
+			for _, tablename := range tablesToSearch {
```
Comment (kritikapradhan) [inline pkg/sloop/webserver/debug.go:137]: can this be tested?

Comment (sana-jawad) [reply pkg/sloop/webserver/debug.go:137]: We have not been testing debug pages as they are purely for debug purpose and not customer facing.

File: pkg/sloop/webserver/webfiles/filter.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ function getUrlVars() {
 function getUrlParam(parameter, defaultvalue){
     var urlparameter = defaultvalue;
     if(window.location.href.indexOf(parameter) > -1){
-        urlparameter = getUrlVars()[parameter];
+        urlparameter = decodeURIComponent(getUrlVars()[parameter]);
```
Comment (kritikapradhan) [inline pkg/sloop/webserver/webfiles/filter.js:19]: Is this related to table change?

Comment (sana-jawad) [reply pkg/sloop/webserver/webfiles/filter.js:19]: Yes its for the same page. The text box in the page gets encoded every time we submit the page

File: pkg/sloop/webserver/debug.go (hunk 1 above)
Comment (kritikapradhan) [reply pkg/sloop/webserver/debug.go:137]: :( 
Should L137 be `for tablename:= range tablesSearch {` ?

**Review by kritikapradhan — APPROVED** [review]
